### ISSUE TITLE
metadata.identifier.resolvable

### ISSUE THREAD

**Issue by gothub** [NCEAS/metadig-checks#60]
## Description

Check if the metadata identifier is resolvable

## Priority

FAIR: Required

## Issues
- this check is contingent on a new DataONE CCI release that contains a fix to the `resolve` service allowing the http `HEAD` request.

## Procedure

- extract the metadata identifier (from metadata). 
- use the DataONE CN `resolve` service with the id to see if DataONE can locate it. If DataONE cannot locate the id, then the check fails.

**Comment by gothub** [comment]
Note that this check is currently disabled in the FAIR suite, until  https://redmine.dataone.org/issues/881 is closed and the fix is incorporated into a DataONE CCI release.

**Comment by gothub** [comment]
Id resolution functionality added in commit d926d1faea61acd9d9a0db59d9849963f23d1a6a
